root-x2.yaml

```diff
@@ -16,7 +16,7 @@ certificate-profile:
   common-name: (FAKE) ISRG Root X2
   organization: Internet Security Research Group
   country: US
-  not-before: 2020-09-04 00:00:00
+  not-before: 2020-09-07 00:00:00
```

Comment:
Ahahah thanks for catching this. This is just because I used `sed` to update the notBefore and notAfter dates in the intermediate config files, and didn't notice that I'd accidentally hit this one too.